src/web/services/timezone.service.spec.ts

```diff
@@ -27,9 +27,9 @@ describe('TimezoneService', () => {
 
   it('should return timezone offsets', () => {
     const tzOffsets: { [key: string]: number } = service.getTzOffsets();
-    expect(tzOffsets['Asia/Singapore']).toEqual(8 * 60);
```

Comment:
Do we want to give similar justification for `GMT-8`?
```
         /*
         * There is a quirk in the ETC/GMT time zones due to the tzdb using POSIX-style signs in the zone names and the
         * output abbreviations. POSIX has positive signs west of Greenwich, while we are used to positive signs east
         * of Greenwich in practice. For example, TZ='Etc/GMT+8' uses the abbreviation "GMT+8" and corresponds to 8
         * hours behind UTC (i.e. west of Greenwich) even though many people would expect it to mean 8 hours ahead of
         * UTC (i.e. east of Greenwich; like Singapore or China).
         * (adapted from tzdb table comments)
         */
```

Reply:
Do we really need to?

Reply:
In case someone don't know why it is `GMT-8`. I am fine with not adding it.